compose.yaml

```diff
@@ -0,0 +1,25 @@
+services:
+  database:
+    image: 'postgres:11.16-alpine3.16'
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_USER: factfinder-user
+      POSTGRES_PASSWORD: factfinder-password
+      POSTGRES_DB: factfinder-local
+    volumes:
+      - ./db-data/:/var/lib/postgresql/data/
+  layers:
+    image: tangoyankee/labs-layers-api:1.3.0
```

Comment:
Were you able to access that docker hub account for the team that you found? If we're going to put this file in, we may as well at least use that account if we can.

Reply:
Will do. Also, I was thinking that it would be better to swap the tag from a semantic version to 'latest'. 
1) This is for local builds and it's safe to point to the last major version
2) Between Bryan, Horatio, and me all changing the layers api last week, I'm up to the fourth semantic version listed here. I'm already over it.